docs/sample_config/default_config.nu

```diff
@@ -198,32 +198,125 @@ let $config = {
   edit_mode: emacs # emacs, vi
   max_history_size: 10000 # Session has to be reloaded for this to take effect
   sync_history_on_enter: true # Enable to share the history between multiple sessions, else you have to close the session to persist history to file
-  menu_config: {
-    columns: 4
-    col_width: 20   # Optional value. If missing all the screen width is used to calculate column width
-    col_padding: 2
-    text_style: green
-    selected_text_style: green_reverse
-    marker: "| "
-  }
-  history_config: {
-    page_size: 10
-    selector: "!"
```

Comment:
I noticed `selector: "!"` is missing from the new configuration below. Does that mean it's not a setting any longer, the functionality was removed, or it's hard-coded now?

Reply:
Unfortunately I had to hard code that value. Couldn't find a way to keep it and make generic menus work.